package.json

```diff
@@ -59,6 +59,7 @@
     "file-loader": "^1.1.11",
     "gateway-addon": "https://github.com/mozilla-iot/gateway-addon-node",
     "greenlock": "^2.2.19",
+    "handlebars": "4",
```

Comment:
One solution is the traditional one. The server starts running as root and then dowgrades itself to the pi user once setup has completed. This should be very clearly delineated to make it obvious that we didn't miss any paths and don't even run the normal server under root.

The second would be to add sudo to the various commands needed in platform.js

Reply:
I already took the second approach of adding sudo to all necessary commands. This does remind me that I haven't checked the sanitization of SSID and PSK. However, any situation it's in now is better than it was before since it's no longer running wpa_cli as root (and doesn't need to)